geode-core/src/main/java/org/apache/geode/internal/cache/InitialImageOperation.java

```diff
@@ -896,8 +895,8 @@ boolean processChunk(List entries, InternalDistributedMember sender, Version rem
         if (diskRegion != null) {
           // verify if entry from GII is the same as the one from recovery
           RegionEntry regionEntry = this.entries.getEntry(entry.key);
-          if (isTraceEnabled) {
-            logger.trace("processChunk:entry={},tag={},re={}", entry, tag, regionEntry);
+          if (isDebugEnabled) {
```

Comment:
I think it's still good to keep those logs in Trace.

Reply:
Restored the trace logging.